dune/stuff/common/vector.hh

```diff
@@ -82,8 +87,10 @@ template< class K >
 struct VectorAbstraction< Dune::DynamicVector< K > >
 {
   typedef Dune::DynamicVector< K > VectorType;
-  typedef K                        ScalarType;
+  typedef typename Dune::FieldTraits< K >::field_type ScalarType;
   typedef ScalarType               S;
+  typedef typename Dune::FieldTraits< K >::real_type RealScalarType;
+  typedef RealScalarType R;
 
```

Comment:
This is one example for indentation. Could you please change these lines to 
```c++
typedef Dune::DynamicVector< K >                    VectorType;
typedef typename Dune::FieldTraits< K >::field_type ScalarType;
typedef typename Dune::FieldTraits< K >::real_type  RealType;
typedef ScalarType S;
typedef RealType   R;
```
so that the indentation of the defined types lines up.